### DISCUSSION TITLE
Rich example

### DISCUSSION THREAD

**Discussion by daniel-bogdoll** [foxglove/community#36] [Q&A]
Hey everybody,

when I take a look at all the available web-visualization tools for autonomous-driving related datasets, Uber AVS is visually the most impressive one regarding the first impression (even though it is unable to process live data etc.). On your webpage you state "Visualize point clouds, overlay bounding boxes, and classification labels" - features which are directly demonstrated on the Uber AVS website and not seen elsewhere, even Cruise only displays raw point clouds in their demo.

Simply to demonstrate the capabilities of your tool I believe it would be great to prepare a demo that shows raw data with object detections on lidar, radar and camera data, maybe with an hd map and additional meta-data as velocity and steering-angle. Maybe nuScenes would be a good start as a database?

Such an example would obviously also give a great entrypoint for beginners if they want to visualize their AD-related algorithms.

**Comment by amacneil** [comment]
Thanks! It's a good point - we would love to demonstrate more features, but we just haven't found any open source datasets with high quality bounding boxes & classifications yet. I will check out nuScenes, that looks interesting (edit: it is licensed for non commercial use only, so I don't think we can use that for screenshots).

Of course, if anyone is willing to share a short segment of data with us for this purpose we would love to chat 😄

**Reply by daniel-bogdoll** [reply]
Additional idea: Since prediction is also a hot topic right now, predicted trajectories within the visualization would also be neat

**Comment by daniel-bogdoll** [comment]
Here I found a small list including licenses...https://www.datasetlist.com/

These might fit:
- https://scale.com/open-datasets/pandaset
- https://www.a2d2.audi/a2d2/en.html
- https://github.com/Robotics-BUT/Brno-Urban-Dataset
- http://apolloscape.auto/

Alternatively, you could just use some open source dataset and create the bounding boxes etc yourself with a SotA algorithm. Would be an even more realistic use case demonstration and it's also fairly easy nowadays.

**Reply by amacneil** [reply]
Thanks! Many of these seem to be static scenes rather than videos, so we will probably need to do a bit of work to support their data formats. But will take a look.

**Comment by daniel-bogdoll** [comment]
https://foxglove.dev/blog/visualizing-nuscenes-with-foxglove-studio --> cool to see that something happened here! :)

**Reply by amacneil** [reply]
Yep - took a while but we got there! Now we just need to update various screenshots on our website.

**Comment by fei4xu** [comment]
Any downloadable data for quick start?

I have downloaded Foxglove and installed, I am looking for some data to run.

**Reply by fei4xu** [reply]
http://wiki.ros.org/loam_velodyne#Datasets

I found some lidar bag files, which can run directly inside Foxglove

**Reply by jtbandes** [reply]
Try out this button in the app:
![EF1C2EF8-1FB3-47FD-A6C8-116EA8624BAF](https://user-images.githubusercontent.com/14237/167729054-adde9ae5-5409-4720-b9d6-72660f9995b0.jpeg)
